File: src/1/1.rb

def multiple(n)
    arr=[]
    (0..n-1).each do |i|
        if i%3==0 or i%5==0
            arr.append(i)
        end
    end
    # puts(arr)
    return arr.reduce(:+)
    
end

puts(multiple(10))
puts(multiple(1000))

File: src/1/1.py
def multiple(n):
    #count=0
    arr=[]
    for x in range(1,n) :
        if x%3==0 or x%5==0:
            # count+=1      
            arr.append(x)

    return arr

print(multiple(10))
print(sum(multiple(10)))


print(multiple(1000))
print(sum(multiple(1000)))
